README.rst

```diff
@@ -3,59 +3,46 @@
 
 Esper
 =====
-**Esper is a lightweight Entity System for Python, with a focus on performance.**
-
-Esper is an MIT licensed Entity System, or, Entity Component System (ECS).
-The design is based on the Entity System concepts outlined by Adam Martin in his blog at
-http://t-machine.org/, and others. Efforts were made to keep it as lightweight and performant
-as possible.
-
-There is a fairly accurate writeup describing Entity Systems in this Wikipedia article:
-https://en.wikipedia.org/wiki/Entity_component_system
-
-Inspired by Sean Fisk's **ecs** https://github.com/seanfisk/ecs,
-and Marcus von Appen's **ebs** https://bitbucket.org/marcusva/python-utils.
+Esper is an MIT licensed `Entity Component System <https://en.wikipedia.org/wiki/Entity_component_system>`_ (ECS) designed to be as lightweight and performant as possible.
+Based on the Entity System concepts outlined by `Adam Martin <http://t-machine.org>`_, `Sean Fisk <https://github.com/seanfisk/ecs>`_, and `Marcus von Appen <https://bitbucket.org/marcusva/python-utils>`_, alongside others.
 
 
```

Comment:
For posterity reasons, I don't think it's a good idea to change the previous release notes.
If you don't mind, can you revert this part?  I usually delete the older notes from the README when releasing a new version anyway, so these will dissapear in time.